docs/serverless-v2/06-01-function-cr.md

```diff
@@ -52,12 +76,42 @@ This table lists all the possible properties of a given resource together with t
+| **spec.source** | Yes | Provides the lambda's source code. |
+| **status.conditions.lastTransitionTime** | Not applicable | Provides a timestamp for the last time the lambda's Pod transitioned from one condition to another. |
+| **status.conditions.message** | Not applicable | Describes a human-readable message on the CR processing progress, success, or failure.  |
+| **status.conditions.reason** | Not applicable | Provides information on the Function CR processing success or failure. See the [**Reasons**](#status-reasons) section for the full list of possible status reasons and their descriptions. |
+| **status.conditions.status** | Not applicable | Describes the status of processing the Function CR by the Function Controller. It can be `True` for success, `False` for failure, or `Unknown` if the CR processing is still in progress. If the status of all conditions is `True`, the overall status of the Function CR is ready. |
+| **status.conditions.type** | Not applicable | Describes a substage of the Function CR processing. There are three condition types that a lambda has to meet to be ready: `Running`, `ConfigurationReady`, `BuildReady`, and `Running`. When displaying the lambda status in the terminal, these types are shown under `CONFIGURED`, `BUILT`, and `RUNNING` columns respectively. All condition types can change asynchronously depending on a type of lambda modification, but all three need to be in the `True` status for the lambda to be considered successfully processed. |
+
+### Status reasons
+
+Processing of a Function CR can succeed, continue, or fail for one of these reasons:
+
+| Reason | Type | Description |
+| --------- | ------------- | ----------- |----------- |
+| `ConfigMapCreated` | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition. |
+| `ConfigMapUpdated` | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies. |
+| `ConfigMapError` | `ConfigurationReady` | The ConfigMap couldn't be created or updated due to an error. |
+| `JobFailed` | `BuildReady` | The image with the lambda's configuration couldn't be created due to an error. |
+| `JobCreated` | `BuildReady` | The Kubernetes Job resource that builds the lambda image was created. |
+| `JobRunning` | `BuildReady` | The Job is in progress.  |
+| `JobsDeleted` | `BuildReady` | Jobs containing the lambda's configuration were deleted. |
+| `JobFinished` | `BuildReady` | The Job was finished and the lambda's image was uploaded to the Docker Registry. |
+| `ServiceCreated` | `Running` | A new KService referencing the lambda's image was created. |
+| `ServiceUpdated` | `Running` | The existing KService was updated after changes in the lambda's image, scaling parameters, variables, or labels. |
+| `ServiceFailed` | `Running` | The KService couldn't be created or updated due to an error. |
```

Comment:
It is rather the service failed - for example, lambda pod is crashing or cannot start, etc.